holidays/countries/israel.py

```diff
@@ -27,15 +39,18 @@
 
 class Israel(ObservedHolidayBase):
     country = "IL"
-    observed_label = "%s (Observed)"
+    default_language = "he"
+    # %s (Observed).
+    observed_label = tr("(נצפה) %s")
```

Comment:
I imagine you got this from the "observed" translation? I'm not sure if this is the right word honestly..

Reply:
That's right. Is current version translation somewhat irrelevant? I'd accept your best shot for this if any.